### DISCUSSION TITLE
Publish report in Bamboo pipeline

### DISCUSSION THREAD

**Discussion by VigneshwaranSooriyamoorthy** [LironEr/cypress-mochawesome-reporter#117] [General]
Is it okay to save combined json report, so it will help in controlling Bamboo pipeline based on mocha task?
I might technically incorrect, if so please let me know how we can PASS/FAIL bamboo pipeline based on cypress-mochawesome-report html

**Comment by VigneshwaranSooriyamoorthy** [comment]
With the help of 'cypress-multi-reporters' and 'mocha-junit-reporter', I managed to publish test result in bamboo pipeline.

Suggestion
It will be handy if 'cypress-mochawesome-reporter' also generates combined junit report.

**Reply by irapths** [reply]
JSON report can be produced by set `mochawesome` option `"json": true`.
!!! This is an answer by @LironEr (I think) I read somewhere but I don't remember where in order to add the link.

As for the Bamboo pipeline I think you have to use XML jUnit style report.
So the `mocha-junit-reporter` is needed in combination with `cypress-multi-reporters` and `cypress-mochawesome-reporter` as you did.
